src/apps/base-images/gdal/Dockerfile

```diff
@@ -0,0 +1,30 @@
+ARG TAG=latest
+#FROM geoservercloud/gs-cloud-base-jre:$TAG as builder
```

Comment:
I assume we can remove this line?

Reply:
yes, it's possible to remove it. Just kept it to remind that at some point we will use the gs-cloud-base-jre  image (@groldan  will be applying some changes while integrating into the CI pipelines)